docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "2.1"
+services:
+  fxa-oauth-console:
+    build:
+      context: .
+      dockerfile: Dockerfile-build
+    command: node bin/server.js
+    image: mozilla/fxa-oauth-console
+    ports:
+      - "10137:10137"
```

Comment:
After looking more into it, it seems to have no problem connecting to the other pm2 servers locally with just the default networking mode. However, I'm getting ```Bad request - state cookie doesn't match``` when I try to login with a created email.